test/integration/create-resource/index.js

```diff
@@ -39,6 +41,20 @@ describe("", () => {
               expect(res.body.extra).to.be.undefined;
             });
 
+            let record;
+            it("should camelize attributes", (done) => {
+              mongoose.models.Organization.findById(createdId, (err, org) => {
```

Comment:
I'd really rather not have direct mongoose queries in these tests, which are just meant to test the output as it's exposed to the user. (They're end to end tests more than integration tests.) Putting mongoose queries here becomes very problematic if, for instance, the user is using a different adapter. So, instead, can we just have a test that creates the resource with the dasherized name, and then tries to fetch it, and checks that the value exists and is as expected? 

In theory, that test could pass and the field could've been stored in mongo under the dasherized key; in practice, because Mongoose won't save document members that aren't part of the schema, there's no way the value will save unless it's been camelized behind the scenes to match the schema's naming. If that's not enough of an assurance, we could test this (I guess) in a unit test for the API controller—mocking the adapter and checking that its `create` method gets called with the properly-camelized names.